community/google-authenticator/APKBUILD

```diff
@@ -39,4 +39,9 @@ package() {
 		> "$pkgdir/etc/pam.d/google-authenticator"
 }
 
-sha512sums="8bf81beaf705f0b12f0ed947960dbbc8155add6d83878b651dc1d6658d520dfbbdbe8085b1f244bcfa06d0dd4dfb4cd2177c0a27df4aab5fd1bc4308eef3f7b0  google-authenticator-1.02.tar.gz"
+check() {
+	cd "$builddir"
+	make test
```

Comment:
Just nitpick `test` is fallback to `check`

Reply:
I replace it with `make check` now.